boards/arm/beaglebone_ai64/bbai_64.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
```

Comment:
The files in the board directory need to match the name of the parent directory, from what I remember.

So instead of bbai_64 use beaglebone_ai64 it vice versa

Reply:
Changing board directory beaglebone_ai64 to bbai_64